docker/federation/lemmy_alpha.hjson

```diff
@@ -22,7 +22,7 @@
   }
   captcha: {
     enabled: false
-    difficulty: medium
+    difficulty: easy
```

Comment:
This doesnt matter at all, the captcha is already disabled above and anyway its only for testing.